### PR TITLE
Bug: Internal links on home page not working

### PR DESCRIPTION
**Pull request by guks**
1. added [ScrollHandler](https://stackoverflow.com/a/56250408/1503444) component to handle hash links
2. added hash IDs to info sections

### CODE REVIEW

File: src/components/ScrollHandler/ScrollHandler.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return null;
+};
+
+ScrollHandler.propTypes = {
+  location: PropTypes.shape({
+    hash: PropTypes.string,
+  }).isRequired,
+};
+
+export default withRouter(ScrollHandler);
```
Comment (jkettmann) [inline src/components/ScrollHandler/ScrollHandler.js]: There's a hook `useLocation` as well. Maybe better to use that for consistency

File: src/components/ScrollHandler/ScrollHandler.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+const ScrollHandler = ({ location }) => {
```
Comment (jkettmann) [inline src/components/ScrollHandler/ScrollHandler.js]: Fyi: There is also the package `react-router-hash-link`. Any reason in particular why you didn't use that? Seems to be the first answer on stackoverflow

**Review by jkettmann — DISMISSED** [review]
Works fine. I leave it up to you if you want to change it or not

File: src/App.js [hunk 3]
```diff
@@ -15,15 +15,14 @@ import Footer from './components/Footer/Footer';
...
-import ScrollHandler from './components/ScrollHandler/ScrollHandler';
 
 function App() {
   return (
     <Theme>
       <AppWrapper>
         <GlobalStyle />
         <Router>
-          <ScrollHandler />
+          {/*           <ScrollHandler /> */}
```
Comment (jkettmann) [inline src/App.js]: Please remove the comment.

**Review by jkettmann — CHANGES_REQUESTED** [review]
Just a leftover comment

File: src/App.js (hunk 3 above)
Comment (guks) [reply src/App.js]: Done

**Review by jkettmann — APPROVED** [review]

**Pull request merged**